queries/collateral/useCollateralShortDataQuery.ts

```diff
@@ -27,25 +33,29 @@ const useCollateralShortDataQuery = (
 		QUERY_KEYS.Collateral.ShortIssuanceFee,
 		async () => {
 			if (currencyKey == null) {
+				return collateralShortNoData;
+			}
+
+			try {
+				const currencyKeyBytes32 = synthetix.js?.toBytes32(currencyKey);
+
+				const [issueFeeRate, shortRate, shortingRewards] = await Promise.all([
```

Comment:
this would retrieve the queries faster and also disable the one that always fails...

Reply:
why are you disabling the short rate?

Reply:
I'll merge this now since it blocks `issueFeeRate` and `shortingRewards` from being displayed. We need to investigate why this happens.